### ISSUE TITLE
Is it supporting Android Pie?

### ISSUE THREAD

**Issue by mnabilurrehman** [dji-sdk/Mobile-UXSDK-Android#60]
I am trying to run it on mobile phones having Android Pie and its crashing whereas it works smooth on lower version of android phone

**Comment by mnabilurrehman** [comment]
Yes! I have tried it on unmodified version of UXSDK. And on multiple phones having android pie.
Nokia 7 plus

**Comment by a2hill** [comment]
I also am having this issue. I have submitted a ticket to DJI support and Lisa Fedane said she was forwarding it on to the engineering team. This was about a week ago.

**Comment by mnabilurrehman** [comment]
> I also am having this issue. I have submitted a ticket to DJI support and Lisa Fedane said she was forwarding it on to the engineering team. This was about a week ago.

Dear @a2hill please when you get any response from DJI Support please update me with the solution as well. This is very critical for me. This will be a nice favor. Thanks in advance.

**Comment by dji-dev** [comment]
Public comment from Lisa Fedane in <a href='https://djisdksupport.zendesk.com/agent/tickets/28837'>Zendesk ticket #28837</a>:<br><br>Can everyone list out the devices that they experience the crash on.  We have run the UXSDK  4.11 Sample App on our Samsung tablet running Android 9 but did not reproduce the crash.

Lisa Fedane
DJI Developer Support

**Comment by a2hill** [comment]
Sorry that is my mistake, our crashes were on Android 10 (Pixel 2 XL, Samsung S10)

**Comment by dji-dev** [comment]
Public comment from Lisa Fedane in <a href='https://djisdksupport.zendesk.com/agent/tickets/28837'>Zendesk ticket #28837</a>:<br><br>Thanks for the correction.  The crash with Android 10 on Pixels 2 and 3 are known bugs that will have a fix released very soon - sorry for the inconvenience and please keep an eye open for an upcoming patch

Lisa Fedane
DJI Developer Support
